### DISCUSSION TITLE
How to disable remote cache for `ios_application`

### DISCUSSION THREAD

**Discussion by kkpattern** [bazelbuild/rules_apple#1640] [Q&A] [answered]
Hi, I want to disable remote-cache for `ios_application`.  Adding `no-remote-cache` to `tags` doesn't work:

```python
ios_application(
    name = "app",
    tags = ["no-remote-cache"],
)
```

What's the proper way to disable remote-cache for `ios_application`?

Thanks.

**Comment by brentleyjones** [comment, marked as answer]
That should work if you have [`--experimental_allow_tags_propagation`](https://bazel.build/reference/command-line-reference#flag--experimental_allow_tags_propagation) enabled. If you want to apply it more globally you can use [`--modify_execution_info`](https://bazel.build/reference/command-line-reference#flag--modify_execution_info):
```
build --modify_execution_info=^(BundleApp|BundleTreeApp)$=+no-remote-cache
```

**Reply by kkpattern** [reply]
Thanks! `--modify_execution_info` works. I don't know would `--experimental_allow_tags_propagation` have some side effects on our project. So I will use `--modify_execution_info` for now.
